assets/scss/base/_helpers.scss

```diff
@@ -177,3 +177,18 @@
 	@extend %no-bullet;
 }
 
+%break-word{
+    -ms-word-break: break-all;
+        word-break: break-all;
+
+        word-break: break-word;
```

Comment:
`break-word` isn't a value for `word-break` according to the spec. This line should be removed?